prost-build/src/lib.rs

```diff
@@ -105,6 +105,20 @@
 //!
 //! If `PROTOC_INCLUDE` is not found in the environment, then the Protobuf include directory bundled
 //! in the prost-build crate is be used.
+//!
+//! ### NixOS related hints
+//!
+//! On NixOS, it is better to specify the location of `PROTOC` and `PROTOC_INCLUDE` explicitly.
+//!
+//! ```bash
+//! $ export PROTOC=/nix/store/<SHA>-protobuf/bin/protoc
```

Comment:
I noted that these lines were useful in a `.nix` related build file as well:

```
        PROTOC = "${pkgs.protobuf}/bin/protoc";
        PROTOC_INCLUDE = "${pkgs.protobuf}/include";
```

Reply:
Oh, thank you! :slightly_smiling_face:  
Should I add those instead? in the docs as well or instead?

Reply:
Is there a `nix` command users can use to get the path of the protoc package in the store, so they don't need to discover the hash?

Reply:
Oh I see! I do not know about that. I should check for `nix-env` maybe? But in my `shell.nix` I do have 

```
{ pkgs ? import <nixpkgs> {} }:
with pkgs;
mkShell {
  buildInputs = [
    python38Full
    stdenv
    stdenv.cc.cc.lib
    python38Packages.tox
    protobuf
  ];
  shellHook = ''
    export PYTHONPATH=`pwd`/$VENV/${python.sitePackages}:$PYTHONPATH;
    export LD_LIBRARY_PATH="${stdenv.cc.cc.lib}/lib64:$LD_LIBRARY_PATH";
    export PROTOC="${pkgs.protobuf}/bin/protoc";
    export PROTOC_INCLUDE="${pkgs.protobuf}/include";
  '';
}
```

Reply:
Ah! I think this is it 

```
$ nix-env -q protobuf --out-path

protobuf-3.7.1  /nix/store/5kl6rzlxa6g86ks9ldl9hm69fd6fl963-protobuf-3.7.1
```